src/service.ts

```diff
@@ -240,6 +240,12 @@ export class DebugService implements IDebugger.IService {
     });
   };
 
+  private onContinued() {
+    this._model.linesCleared.emit();
+    this._model.sidebar.callstack.model.frames = [];
+    this._model.sidebar.variables.model.scopes = [];
```

Comment:
This should be replaced with
```typescript
    this._model.callstackModel.frames = [];
    this._model.variablesModel.scopes = [];
```

Reply:
Yes indeed, first rebase wasn't on #100 :)